app/model/media/AudioModel.js

```diff
@@ -558,17 +594,52 @@ SDL.AudioModel = Em.Object.extend({
       }
     },
 
+    transitionInBackground(nextState){
```

Comment:
```suggestion
    changeAudioSource(nextState, isBackground) {
      if (!isBackground) {
        SDL.States.goToStates(nextState);
        return;
      }
```
Looks like we could move some of this logic into this function to make it more generic